src/main/java/com/uber/jenkins/phabricator/coverage/CoberturaXMLParser.java

```diff
@@ -66,6 +82,18 @@
             try {
                 is = new FileInputStream(file);
                 db = dbf.newDocumentBuilder();
+                db.setEntityResolver(new EntityResolver() {
```

Comment:
Remove anonymous inner class.

Make the EntityResolver a private static final constant that is initialized at the beginning